app/addons/dashboard/components.react.jsx

```diff
@@ -0,0 +1,190 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'react',
+  'addons/dashboard/stores'
+], function (App, FauxtonAPI, React, Store) {
+
+  var activeTaskList = Store.dashboardStore;
+
+  var DashboardController = React.createClass({
+
+    getStoreState: function () {
+      return {
+        collection: activeTaskList.getCollection(),
+
+        setPolling: activeTaskList.setPolling,
```

Comment:
@benkeen @robertkowalski 
Can you explain little bit more about this?  I just follow the implementation of ActiveTaskWidget. So I am not clear about what you are suggesting. If you can, it will be very helpful specially for future works.